pkg/loop/internal/pb/chain_reader.proto

```diff
@@ -34,6 +35,13 @@ message QueryKeyRequest {
   LimitAndSort limit_and_sort = 3;
 }
 
+// ReplaySequenceRequest has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.ChainReader.ReplaySequence].
+message ReplaySequenceRequest {
+  string contract_name = 1;
+  string key = 2;
+  string blockID = 3;
```

Comment:
```suggestion
  string block_id = 3;
```

Reply:
Resolved

Reply:
Oh interesting. I guess it can't detect acronyms. Is there a way to override?

Reply:
looks like its possible by using  protoc-gen-gogoslick